packs/git/rules/github-pr-merge.json

```diff
@@ -9,16 +9,7 @@
         }
     },
 
-    "criteria": {
```

Comment:
I assume you have intentionally removed this section?

It looks like previously we sent all the github events to this webhook and now we only send merge events to this webook?